charts/consul/test/unit/server-acl-init-cleanup-job.bats

```diff
@@ -70,3 +70,48 @@ load _helpers
       yq 'length > 0' | tee /dev/stderr)
   [ "${actual}" = "true" ]
 }
+
+#--------------------------------------------------------------------
+# global.acls.tolerations and global.acls.nodeSelector
+
+@test "serverACLInitCleanup/Job: tolerations not set by default" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/server-acl-init-job.yaml  \
```

Comment:
These should be testing `server-acl-init-cleanup` template files

Reply:
Oh geez, that's embarrassing. Thank you for catching the copypasta issue.